### PR TITLE
Add Xbox builders

### PR DESCRIPTION
**Pull request by kaidokert**
Adds build configs to Xbox

b/266878119

### CODE REVIEW

**Review by TyHolc — COMMENTED** [review]
Based on the build failures it looks like this might be missing either a UWP SDK or Xbox SDK

**Comment by kaidokert** [comment]
> Based on the build failures it looks like this might be missing either a UWP SDK or Xbox SDK

Because it's still being executed in the wrong container - i put a todo / comment in the JSON file. I'd appreciate if someone can pick this up and land it - i won't make much forward progress in the next few days.

File: .github/workflows/xb1.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: win32
```
Comment (isarkis) [inline .github/workflows/xb1.yaml:1]: this should be xb1.

Comment (kaidokert) [reply .github/workflows/xb1.yaml]: Thanks, fixed

**Pull request closed without merging**